tmt/steps/prepare/feature/epel-enable.yaml

```diff
@@ -20,8 +20,11 @@
           register: output
           changed_when: output.rc != 0
 
-    - name: Enable EPEL and EPEL-Next repos on RHEL 8+
-      when: ansible_distribution == "RedHat" and ansible_distribution_major_version | int >= 8
+    - name: Enable EPEL and EPEL-Next repos on RHEL 8, 9
+      when:
+        - ansible_distribution == "RedHat"
+        - ansible_distribution_major_version | int >= 8
+        - ansible_distribution_major_version | int < 10
```

Comment:
I would dare to say that there will be EPEL10, its nonexistence is just temporary. Given that, I would treat it as a temporary exception, i.e. no change to the task name, and a comment letting the reader know this will pass. What do you think about the following?

```yaml
when:
  - ansible_distribution == "RedHat"
  - ansible_distribution_major_version | int >= 8
  # Drop the following once EPEL10 becomes available
  - ansible_distribution_major_version | int != 10
```